test/docker-compose.yml

```diff
@@ -85,6 +85,20 @@ services:
       - ./support:/support
       - node_modules:/node_modules
 
+  test-acceptance.testcafe:
```

Comment:
This test should be added to https://github.com/Codeception/CodeceptJS/blob/master/.circleci/config.yml#L7 so we could add it to the pipeline